src/tests/ftest/get_remote_files.sh

```diff
@@ -153,10 +153,12 @@ scp_files() {
         archive_name="${file_name%%.*}.$(hostname -s).${file_name#*.}"
         if scp -r "${file}" "${2}"/"${archive_name}"; then
             copied+=("${file}")
-            if ! rm -fr "${file}"; then
-                echo "  Error removing ${file}"
-                echo "    $(ls -al ${file})"
-                rc=1
+            if [[ ! "${file}" =~ "test.cov" ]]; then
```

Comment:
(lint) Don't quote right-hand side of =~, it'll match literally rather than as a regex. [SC2076]